.eslintrc

```diff
@@ -35,7 +35,8 @@
       }
     ],
     "import/no-default-export": "error",
-    "no-else-return": "error"
+    "no-else-return": "error",
+    "no-labels": "off"
```

Comment:
what's does this change do? A separate PR?

Reply:
Allows [labels](https://eslint.org/docs/rules/no-labels) to be used

it makes the swap search loop more readable imo